src/components/form/Form.js

```diff
@@ -0,0 +1,204 @@
+
+    // __________________________________inputs and errors
+    const [inputs, setInputs] = useState({
+        name: '',
+        email: '',
+        request: ''
+    })
+
+    const [errors, setErrors] = useState({
+        name: '',
+        email: ''
+    });
+
+    const errorMessage = '(This field is required)'
+
+    const checkFormValidity = () => {
+        setIsValid(formRef.current.checkValidity())
+    }
+
+    // _____________________________ determine input for validation
+    const pickInput = (name, valid) => {
+        if (name === 'name') {
+            if (valid) {
+                setNameValid(true)
+                return
+            }
+            else setNameValid(false)
+        }
+        if (name === 'email') {
+            if (valid) {
+                setEmailValid(true)
+                return
+            }
+            else setEmailValid(false)
+        }
+    }
+
+    const handleChange = (event) => {
+        const { id, value } = event.target;
+        errors[id] && updateErrors(event);
+        setInputs({
+            ...inputs,
+            [id]: value
+        });
+    }
+
+    const updateErrors = (event) => {
+        const { id } = event.target;
+        const valid = event.target.validity.valid;
+        pickInput(id, valid)
+        setErrors({
+            ...errors,
+            [id]: errorMessage
+        });
+    }
+
+    // ________________________________________reset inputs
+    const resetInputs = () => {
+        setInputs({
+            name: '',
+            email: '',
+            request: ''
+        })
+        setErrors({
+            name: '',
+            email: ''
+        })
+        setIsValid(false);
+        setNameValid(true);
+        setEmailValid(true);
+    }
+
+    // ___________________reset inputs when popup is closed
+    useEffect(() => {
+        resetInputs();
+    }, [props.open]);
+
+    // ____________________________close popup
+    const closePopup = () => {
+        props.closePopup();
+    }
+
+    // ___________________________________ Google form data
+    const formName = 'entry.1259736291';
+
+    const formEmail = 'entry.1503900032';
+
+    const formRequest = 'entry.1763891937';
+
+    const formURL = 'https://docs.google.com/forms/d/18EfQ-4I_6IffAlGivd0wxGbebGZ1REUGuVuJe4_YvgE/formResponse';
+
+    let submittedForm = false;
+    let hiddenFrameLoaded = false;
+
+    function handleHiddenFrameLoaded() {
+        if (hiddenFrameLoaded) {
+            submittedForm = true;
+        } else {
+            hiddenFrameLoaded = true;
+        }
+    }
+
+    return (
+        <Overlay open={props.open}>
+            <Wrapper>
+                <Exit
+                    src={exit}
+                    onClick={closePopup} />
+                <Title>
+                    Leave your contacts and request,
+                    and our manager will reach out to you to
+                    establish our futher parntership!
```

Comment:
'partnership' instead of 'parntership'
'further' instead of 'futher' (? guessing on this one)

Reply:
I think it should actually be future.  That makes way more sense in context

Reply:
yeah that makes sense